lib/vector-tap/src/lib.rs

```diff
@@ -95,20 +96,34 @@ impl EventFormatter {
     }
 }
 
+#[derive(Clone, Debug)]
+pub enum OutputChannel {
+    Stdout(EventFormatter),
+    AsyncChannel(tokio_mpsc::Sender<Vec<GraphQLTapOutputEvent>>),
+}
+
 /// Error type for DNS message parsing
-#[derive(Debug, Snafu)]
+#[derive(Debug)]
 pub enum TapExecutorError {
-    #[snafu(display("[tap] Couldn't connect to API via WebSockets"))]
-    ConnectionFailure,
+    ConnectionFailure(tokio_tungstenite::tungstenite::Error),
     GraphQLError,
 }
 
+impl fmt::Display for TapExecutorError {
```

Comment:
Nit: Remove this implementation since we are using only the `Debug` trait with `{:?}`.